pkg/test-infra/tidb/operation.go

```diff
@@ -459,6 +467,32 @@ func getTidbDiscoveryService(tc *v1alpha1.TidbCluster) *corev1.Service {
 	}
 }
 
+func parseConfig(config string) (string, error) {
+	// Parse config
+	configData, err := readFileAsString(config)
+	if err == nil {
+		return configData, nil
+	}
+	// parse config string like "base64://BASE64CONTENT"
+	// Supported scheme: base64
+	u, err := url.Parse(config)
+	if err != nil {
+		return "", err
+	}
+	switch u.Scheme {
+	case "base64":
+		// An workaround: When parse url like "base64://BASE64CONTENT", BASE64CONTENT would be parsed in url.Host
+		configData, err = readBase64AsString(u.Host)
+		if err != nil {
+			return "", err
+		}
+	default:
+		return "", fmt.Errorf("not a supported scheme: %s\nsupported schemes like base64://BASE64CONTENT", config)
```

Comment:
Generally, we do not insert `newline` in error message.

Reply:
Have removed it.